src/schemas/Clinical/index.ts

```diff
@@ -663,31 +734,6 @@ const resolvers = {
       return response ? true : false;
     },
   },
-  ClinicalData: {
```

Comment:
So the default sort order is to have clinical entity data sorted so that records with clinical errors are sorted to the top. This creates a relationship between clinicalEntities and clinicalErrors, which are separate records from separate DBs. `clinicalData` resolver must now request both `clinicalEntities` and `clinicalErrors` from the service on every query, in order to sort error data to the top.

Currently the FE needs both Clinical Entity & Clinical Error values every time it requests Clinical Data. Pagination limits mean we will never request/return more than 100 entities + 100 errors at any time. Errors are requested by DonorId, using subset of IDs from Clinical Entities.

Ideally we can make the API more single responsibility w/ nested resolvers, but I'm not sure how. @joneubank and I previously separated Entities + Errors by using a nested resolver (Clinical Entities > Clinical Errors). However this creates a parent > child relationship, and Clinical Errors cannot sort the data from Clinical Entities.

Alternately we revert, keep API resolvers single responsibility, and sort by errors on the front end.

Reply:
Reverting to Error Sort on FE for now, but I've added a few Types + small improvements through code review 
Saved API Sorting code in the event we align on double requests / nested resolver approach